Dockerfile

```diff
@@ -1,16 +1,23 @@
-FROM alpine:3.9
+FROM alpine:3.20 AS ssm-builder
+
+RUN apk add dpkg curl; \
+    curl "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/ubuntu_64bit/session-manager-plugin.deb" -o "session-manager-plugin.deb"; \
+    dpkg -x session-manager-plugin.deb session-manager-plugin
+
+
+FROM alpine:3.20
 
 RUN apk add --update --no-cache \
     curl \
     jq \
     ca-certificates \
     bash \
-    python \
-    && python -m ensurepip \
-    && rm -r /usr/lib/python*/ensurepip \
-    && pip install --upgrade pip setuptools \
-    awscli --ignore-installed \
-    && rm -r /root/.cache
+    aws-cli \
+    gcompat \
```

Comment:
I think we can also remove `gcompat` if the reason for pulling this is the manually installed session-manager-plugin